### PR TITLE
Get accessKey from account and Add log

### PR DESCRIPTION
**Pull request by woomurf**

### CODE REVIEW

File: src/auth.ts [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import AinftBase from './ainftBase';
 import { HttpMethod, User } from './types';
 export default class Auth extends AinftBase {
-  async initializeApp(appId: string, userId: string, accessKey: string): Promise<void> {
+  async initializeApp(appId: string, userId: string): Promise<void> {
+    console.log('Start app initializing... This may take up to a minute.');
```
Comment (liayoo) [inline src/auth.ts:5]: minor) Starting app initialization...

**Review by liayoo — APPROVED** [review]
LGTM with a comment. Thanks!

File: src/auth.ts (hunk 1 above)
Comment (woomurf) [reply src/auth.ts]: Thanks!

**Comment by woomurf** [comment]
Thanks for the fast review!

**Review by ehgmsdk20 — COMMENTED** [review]
LGTM, thanks!

**Pull request merged**